src/app/components/header/header.component.tsx

```diff
@@ -6,25 +9,53 @@ interface HeaderProps {
 }
 
 function NavBar(props: any) {
+    const menuVisibility = false;
```

Comment:
remove menuVisibility, use simply false as initial value for useState hook

Reply:
menuVisibility constant removed, used false as initial value for useState hook